src/Tribe/Views/V2/Hooks.php

```diff
@@ -299,13 +301,48 @@ public function filter_map_canonical_query_args( $map, $url, $rewrite ) {
 	 *
 	 * @return array $classes
 	 */
-	public function filter_body_class( $classes ) {
-		$classes = $this->container->make( Theme_Compatibility::class )->filter_add_body_classes( $classes );
+	public function filter_body_classes( $classes ) {
 		$classes = $this->container->make( Template_Bootstrap::class )->filter_add_body_classes( $classes );
 
 		return $classes;
 	}
 
+	/**
+	 * Add body classes.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function add_body_classes() {
+		$this->container->make( Theme_Compatibility::class )->add_body_classes();
+		$this->container->make( Template_Bootstrap::class )->add_body_classes();
+	}
+
+	/**
+	 * Logic for if body classes should be added.
+	 *
+	 * @since TBD
+	 *
+	 * @param boolean $add   Whether to add classes or not.
+	 * @param string  $queue The queue we want to get 'admin', 'display', 'all'.
+	 *
+	 * @return boolean Whether to add classes or not.
+	 */
+	public function body_classes_should_add( $add, $queue ) {
+		$context = tribe_context();
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space

<sub>hash: 3ea15802f3dec764fa6c069c1ce7deb1</sub>